main/test/flix/Test.Exp.Jvm.InvokeStaticMethod.flix

```diff
@@ -46,7 +46,7 @@ namespace Test/Exp/Jvm/InvokeStaticMethod {
         try {
             of1("end", ["p1", "p2", "p3"])
         } catch {
-            case _: ##java.nio.file.InvalidPathException => unsafe_cast null as ##java.nio.file.Path
+            case _: ##java.nio.file.InvalidPathException => super_cast null
```

Comment:
This should not be a supercast, according to Matt.

Reply:
Ah of course because of the safety issue mentioned in https://github.com/flix/flix/issues/5002